README.md

```diff
@@ -567,6 +567,7 @@ _Language specific game engine development libraries / frameworks / code._
 - C#: Entity Component System
     - 🎉 [Arch](https://github.com/genaray/Arch) - High-performance ECS with optional multithreading.
     - 🎉 [DefaultEcs](https://github.com/Doraku/DefaultEcs) - ECS framework designed for game development.
+    - 🎉 [Friflo.Engine.ECS](https://github.com/friflo/Friflo.Json.Fliox/blob/main/Engine/README.md) - Archetype based ECS with focus 🎯 on performance, cache locality and DX.
```

Comment:
Looks like this project uses the copyleft AGPL license. This list uses the Locked Emoji (🔒) for copyleft projects, the Party Popper Emoji (🎉) is for projects with [permissive licenses](https://en.wikipedia.org/wiki/Permissive_software_license).